### DISCUSSION TITLE
Прототипы для выживания на планете.

### DISCUSSION THREAD

**Discussion by VigersRay** [space-sunrise/space-station-14#130] [Ideas]
Нам категорически нужны прототипы позволяющие на пустой планете из говна и палок собирать что-то интересное.
На той же дельте всегда есть портал, и люди там пытаются что-то строить, но фиг чё у них нормально получается.

Так вот что нам надо:
- возможность адекватно строить стены без стали.
- возможность строить костры и готовить на них еду. (Тот же гриль, емнип)
- спавн животных. (Это очень легко и не вызовет проблем с нагрузкой так как у нас НПС отключены если рядом нет игроков)
-крафт базового оружия.
-крафт грядок.
-спавн мелких камушек которые можно собирать.
...Остальное предлагайте сами.

Ещё будет очень круто если мы позже сможем реализовать роли для призраков в виде каких-то выживальщиков на изолированной планете, или в идеале вообще открыть растоподобный сервер.

**Comment by andreySS14** [comment]
Лучше иначе - дать возможность делать: печи (как переработчик руд), рандомный спавн некоторых руд как плазма, сталь, уголь, серебро золото и тд, деревянные каркасы (только для деревянных зданий) создание лопатой грядки, могилы и иного, делать костры из досок с возможностью топлива (как генератор) и возможностью поджигать и тушить их, тогда можно начинать
